core/include/traccc/seeding/doublet_finding.hpp

```diff
@@ -14,6 +14,7 @@
 #include "traccc/seeding/detail/spacepoint_type.hpp"
 #include "traccc/seeding/doublet_finding_helper.hpp"
 #include "traccc/utils/algorithm.hpp"
+#include "traccc/utils/logging_mixin.hpp"
```

Comment:
In some files you include both `logging.hpp` and `logging_mixin.hpp`. But somewhere you only include `logging_mixin.hpp`.

What should be the convention? As long as the "logger base class" is expected to pull in the `Logger` type, I think it's fine to only include `logging_mixin.hpp`. We have plenty of examples of that in our other projects. No Athena algorithms ever explicitly include headers related to logging, status codes, and such. As they should by definition be available.

This is not a super important point, but could still be worth thinking about.

Reply:
Fixed; cases where only the mixin can be included have been made to do so.